### DISCUSSION TITLE
Error: Unknown environment names 'tasmota32-sensors'

### DISCUSSION THREAD

**Discussion by ionultd** [arendst/Tasmota#12178] [Q&A] [answered]
When compiling latest firmware there appears to no more tasmota32-sensors. What happened, what did I miss?

**Comment by ascillato** [comment, marked as answer]
It is now included in Tasmota32.bin

**Reply by ionultd** [reply]
Wow thks 4 the prompt reply. So all the sensors are enabled?

**Reply by ascillato** [reply]
You can check the full list of features included in the following:

https://github.com/arendst/Tasmota/blob/development/BUILDS.md

**Reply by sfromis** [reply]
There are lots of sensors where you'll have to build a binary to include them. This is/was also the case with the precompiled -sensors binary.

**Reply by ionultd** [reply]
Yes I usually had to do so by adding the custom use statements to the user_config_override file. Is this still the case? As I need to add a sensor that is not included by default.

**Reply by sfromis** [reply]
No changes there. The only difference is that the additional sensors by default included in tasmota32-sensors are now included in tasmota32. Anyway, the general concept would not be to add to -sensors, but start with the base environment, and add needed stuff there. The tasmota-something environments are just not meant for additions at all.

**Reply by ionultd** [reply]
So does one still enable needed sensors in user_config_override

Does one still choose default_envs in platformio_override

**Reply by sfromis** [reply]
Yes, user_config_override.h as usual. You can choose default environment in platformio_override.h, but you can also specify the environment on the command to build.
